pkg/cmd/inletsoperator_app.go

```diff
@@ -64,28 +70,32 @@ func makeInstallInletsOperator() *cobra.Command {
 
 		os.Setenv("HELM_HOME", path.Join(userPath, ".helm"))
 
-		_, err = tryDownloadHelm(userPath, clientArch, clientOS, false)
+		if helm3 {
+			helm3Version := "v3.0.1"
```

Comment:
We must have this in > 1 place. Let's extract a const in the package

Reply:
Fixed.